src/dashboard/src/templates/administration/processing.html

```diff
@@ -25,6 +25,9 @@ <h3>{% trans "Processing configuration" %}</h3>
           </td>
           <td>
             <a class="btn btn-default btn-sm" href="{% url "components.administration.views_processing.edit" item.1 %}">{% trans "Edit" %}</a>
+            {% if item.1 == "default" or item.1 == 'automated' %}
+            <a class="btn btn-default btn-sm" title="Reset to original values" href="{% url "components.administration.views_processing.reset" item.1 %}">{% trans "Reset" %}</a>
```

Comment:
Nitpick: this may not be very idiomatic but notice how we indent code inside template blocks.